apps/dashboard/config/environments/development.rb

```diff
@@ -60,4 +60,7 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.active_job.queue_adapter = ActiveJob::QueueAdapters::AsyncAdapter.new \
```

Comment:
We just created an `ActiveJobs` (plural) module. I'm guessing we should maybe tweak that to disambiguate from Rails `ActiveJob`?

Reply:
Yes but I don't know what to name it. A new issue.